src/dropbox.js

```diff
@@ -41,7 +41,28 @@
   var hasLocalStorage;
   var AUTH_URL = 'https://www.dropbox.com/1/oauth2/authorize';
   var SETTINGS_KEY = 'remotestorage:dropbox';
-  var cleanPath = RS.WireClient.cleanPath;
+  var PATH_PREFIX = '/remotestorage';
+
+  /**
+   * Function: mapPath(path)
```

Comment:
The documentation mentions a different function name than what's actually used (`mapPath` vs. `getDropboxPath`).